### PR TITLE
Unify Flow and FEA unsteady options + fix unsteady FEA adjoints + add FEA python wrapper examples

### PR DESCRIPTION
**Pull request by pcarruscag**
## Proposed Changes
Use TIME_STEP and MAX_TIME in the FEA solver instead of DYN_*

## PR Checklist
- [X] I am submitting my contribution to the develop branch.
- [X] My contribution generates no new compiler warnings (try with --warnlevel=3 when using meson).
- [X] My contribution is commented and consistent with SU2 style (https://su2code.github.io/docs_v7/Style-Guide/).
- [X] I have added a test case that demonstrates my contribution, if necessary.
- [X] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp), if necessary.

### CODE REVIEW

File: TestCases/py_wrapper/custom_load_fea/run.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+
+import pysu2
+from mpi4py import MPI
+import numpy as np
```
Comment (pcarruscag) [inline TestCases/py_wrapper/custom_load_fea/run.py:29]:
```suggestion
```

File: TestCases/py_wrapper/custom_load_fea/run.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+def main():
+  comm = MPI.COMM_WORLD
+  rank = comm.Get_rank()
```
Comment (pcarruscag) [inline TestCases/py_wrapper/custom_load_fea/run.py:33]:
```suggestion
```

**Comment by kursatyurt** [comment]
Nice clean-up/unification

File: SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp [hunk 3]
```diff
@@ -780,6 +780,8 @@ void CDiscAdjMultizoneDriver::SetAdjObjFunction() {
   }
   if (rank == MASTER_NODE) {
     AD::SetDerivative(ObjFunc_Index, SU2_TYPE::GetValue(seeding));
+  } else {
+    AD::SetDerivative(ObjFunc_Index, 0.0);
```
Comment (pcarruscag) [inline SU2_CFD/src/drivers/CDiscAdjMultizoneDriver.cpp:784]:
```suggestion
```

**Pull request merged**